src/api/schema/index.js

```diff
@@ -4,14 +4,23 @@ import UserAPI from '@api/User'
 import ProjectAPI from '@api/Project'
 import ProjectUserAPI from '@api/ProjectUser'
 import Root from './types/Root.graphql'
+import Scalars from './scalars/scalars.graphql'
+import ScalarsResolvers from './scalars/resolvers'
+import Status from './types/Status.graphql'
 import ValidationError from './types/ValidationError.graphql'
 import User from './types/User.graphql'
 import Project from './types/Project.graphql'
 import ProjectUser from './types/ProjectUser.graphql'
-import Scalars from './scalars/scalars.graphql'
-import ScalarsResolvers from './scalars/resolvers'
 
-const typeDefs = [Root, Scalars, ValidationError, User, Project, ProjectUser]
+const typeDefs = [
+  Root,
+  Scalars,
+  Status,
+  ValidationError,
```

Comment:
Ideally we would export a single `Response` interface but it doesn’t work as expected and there’s a `graphql-tools` warning.